document-store/src/integrationTest/resources/query/test_last_aggregation_operator.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "item":"Shampoo",
```

Nit: Could you please prettify the JSON (specifically spaces before the values)? 
```suggestion
    "item": "Shampoo",
```